### PR TITLE
Remove reference to installing linkerd-base helm chart in 2.12 docs

### PR DESCRIPTION
**Pull request by stewartshea**
Fixes https://github.com/linkerd/website/issues/1529

### CODE REVIEW

File: linkerd.io/content/2.12/tasks/generate-certificates.md [hunk 1]
```diff
@@ -73,20 +73,15 @@ linkerd install \
 ```
 
 Or when installing with Helm, first install the `linkerd-crds` and
-`linkerd-base` charts:
+`linkerd-control-plane` charts:
```
Comment (kleimkuhler) [inline linkerd.io/content/2.12/tasks/generate-certificates.md:76]:
```suggestion
Or when installing with Helm, first install the `linkerd-crds` chart:
```

File: linkerd.io/content/2.12/tasks/generate-certificates.md [hunk 2]
```diff
@@ -73,20 +73,15 @@ linkerd install \
...
 
 ```bash
 helm install linkerd-crds linkerd/linkerd-crds -n linkerd --create-namespace
 
-helm install linkerd-base -n linkerd linkerd/linkerd-base
-```
-
-Then install the `linkerd-control-plane` chart:
-
-```bash
```
Comment (kleimkuhler) [inline linkerd.io/content/2.12/tasks/generate-certificates.md:86]: We should keep these lines as is.

**Comment by wmorgan** [comment]
@stewartshea did you want to address the feedback so that we can merge this?

**Comment by stewartshea** [comment]
Thanks for the heads up, I hadn't noticed the comments. I'll have a look tomorrow as I'm not at my desk today. Thanks so much!

**Comment by stewartshea** [comment]
I believe I've incorporated the suggestions - let me know if there are any other issues or typos. Thanks!

**Review by kleimkuhler — APPROVED** [review]
Thanks!

**Pull request merged**